client/src/components/ContentNavigator/convert.ts

```diff
@@ -0,0 +1,198 @@
+  const now = new Date();
+  const nowString = now.toISOString();
+  const nowTimestamp = String(now.getTime());
+  const arrayIdNode: string[] = [];
+  const flowData = {
+    creationTimeStamp: nowString,
+    modifiedTimeStamp: nowString,
+    createdBy: "user@sas.com",
+    modifiedBy: "user@sas.com",
+    version: 2,
+    id: null,
+    name: outputFile,
+    description: null,
+    properties: {
+      UI_PROP_DF_OPTIMIZE: "false",
+      UI_PROP_DF_ID: "120081fc-2d7f-4cfc-bcd3-47f9684e9763",
+      UI_PROP_DF_EXECUTION_ORDERED: "false",
+    },
+    links: [],
+    nodes: {},
+    parameters: {},
+    connections: [],
+    extendedProperties: {},
+    stickyNotes: [],
+  };
+
+  for (let idx = 0; idx < inputList.length; idx++) {
+    const idNode = `id-${Date.now()}-${Math.floor(Math.random() * 10000)}`;
+    arrayIdNode.push(idNode);
+    const idNote = `id-${Date.now()}-${Math.floor(Math.random() * 10000)}`;
+    const entry = inputList[idx];
+    const propports = getPropPort(idx, inputList);
+    const stepNode = {
+      nodeType: "step",
+      version: 1,
+      id: idNode,
+      name: stepTitle[entry.language],
+      note: {
+        version: 1,
+        id: idNote,
+        name: null,
+        description: null,
+        properties: {
+          UI_NOTE_PROP_HEIGHT: "0",
+          UI_NOTE_PROP_IS_EXPANDED: "false",
+          UI_NOTE_PROP_IS_STICKYNOTE: "false",
+          UI_NOTE_PROP_WIDTH: "0",
+        },
+      },
+      priority: idx,
+      properties: {
+        UI_PROP_COLORGRP: "0",
+        UI_PROP_IS_INPUT_EXPANDED: "false",
+        UI_PROP_IS_OUTPUT_EXPANDED: "false",
+        UI_PROP_NODE_DATA_ID: stepRef[entry.language],
+        UI_PROP_NODE_DATA_MODIFIED_DATE: nowTimestamp,
+
+        UI_PROP_XPOS: String((idx + 1) * 150),
+        UI_PROP_YPOS: "75",
+      },
+      portMappings: [
+        {
+          mappingType: "tableStructure",
+          portIndex: 0,
+          portName: "outTables",
+          tableStructure: {
+            columnDefinitions: null,
+          },
+        },
+      ],
+      stepReference: {
+        type: "uri",
+        path: `/dataFlows/steps/${stepRef[entry.language]}`,
+      },
+      arguments: {
+        codeOptions: {
+          code: entry.code,
+          contentType: "embedded",
+          logHTML: "",
+          resultsHTML: "",
+          variables: [
+            {
+              name: "_input1",
+              value: {
+                portIndex: 0,
+                portName: "inTables",
+                referenceType: "inputPort",
+              },
+            },
+            {
+              name: "_output1",
+              value: {
+                arguments: {},
+                portIndex: 0,
+                portName: "outTables",
+                referenceType: "outputPort",
+              },
+            },
+          ],
+        },
+      },
+    };
+    stepNode.properties = { ...stepNode.properties, ...propports };
```

Comment:
```suggestion
    stepNode.properties = { ...stepNode.properties, ...propPorts };
```

Reply:
changed variable name in cf89c943731e7c2fef458becd0ea636475db0f59